atomics/T1110.003/T1110.003.yaml

```diff
@@ -105,3 +105,59 @@ atomic_tests:
         }
       }
       Write-Host "End of password spraying"
+
+- name: Password spray all Azure AD users with a single password 
+  auto_generated_guid: a8aa2d3e-1c52-4016-bc73-0f8854cfa80a
+  description: |
+    Attempt to brute force all Azure AD users with a single password (called "password spraying") via AzureAD Powershell module.
+    Valid credentials are only needed to fetch the list of Azure AD users.
+  supported_platforms:
+  - azure-ad
+  input_arguments:
+    password:
+      description: Single password we will attempt to auth with (if you need several passwords, then it is a bruteforce so see T1110.001)
+      type: String
+      default: P@ssw0rd!
+    valid_username:
+      description: Valid username to retrieve Azure AD users
+      type: String
+      default: bruce.wayne@contoso.com
```

Comment:
the default value is fine in this line but can you enhance the description of the dependencies which says :
" We encourage users running this atomic to add a valid microsoft account domain; for eg: bruce.wayne@<valid_ms_account.com>" 

This will be very useful for users who are running these atomics to improve their detection coverage